### PR TITLE
Update Canaries to reflect Sourcepoint (cookie banner TP) changes

### PR DESCRIPTION
**Pull request by chrislomaxjones**
When we clear cookies, make sure we also clear localstorage. This is to ensure that any localstorage items stored by CMP are removed.

### CODE REVIEW

File: src/cmp_aus.js [hunk 1]
```diff
@@ -130,6 +135,7 @@ const loadPage = async (page, url) => {
 	// Clear cookies before starting testing, to ensure the CMP is displayed.
 	const client = await page.target().createCDPSession();
 	await clearCookies(client);
+	await clearLocalStorage(page);
```
Comment (arelra) [inline src/cmp_aus.js:138]: Doesn't have to be now but we could call these two functions in a single `teardown()` function that is called by each test.

Comment (chrislomaxjones) [reply src/cmp_aus.js:138]: If it works 🤞

**Review by arelra — APPROVED** [review]

File: src/cmp_aus.js (hunk 1 above)
Comment (chrislomaxjones) [reply src/cmp_aus.js:138]: Thank you @kenoir!

So SP only saves to localstorage in a TCFv2 region, and not for the others?

Comment (chrislomaxjones) [reply src/cmp_aus.js]: @arelra this may mean we can't go with your `teardown` suggestion since we call localstorage later in the canary run.

Comment (kenoir) [reply src/cmp_aus.js]: I believe it also uses LocalStorage in the CCPA jurisdiction, but the thing to note is that you can't clear LocalStorage before you've loaded a page (whereas you can with cookies). You could probably implement a generic resetting function that looks like:

```ts
const resetPage = async (client: CDPSession, page: Page) => {
  await loadPage(page, url);
  await clearCookies(client);
  await clearLocalStorage(page);
  await loadPage(page, url);
}
```

**Review by arelra — APPROVED** [review]
Great work in making this more robust! 👏

**Pull request merged**